.github/workflows/build.yml

```diff
@@ -493,19 +501,28 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Set cache attributes
+        if: "!startsWith(github.ref, 'refs/tags/v')"
+        run: |
+          echo "CACHE_FROM=type=gha,scope=frontend-${{ needs.frontend-base.outputs.typ }}" >>${GITHUB_ENV}
+          echo "CACHE_TO=type=gha,scope=frontend-${{ needs.frontend-base.outputs.typ }}" >>${GITHUB_ENV}
       -
         name: Build ${{ needs.frontend-base.outputs.typ }}/${{ needs.frontend-base.outputs.tag }}
         run: |
           ./frontend/dockerfile/cmd/dockerfile-frontend/hack/release "${{ needs.frontend-base.outputs.typ }}" "${{ needs.frontend-base.outputs.tag }}" "$DF_REPO_SLUG_TARGET" "${{ needs.frontend-base.outputs.push }}"
         env:
           PLATFORMS: ${{ env.PLATFORMS }},linux/386,linux/mips,linux/mipsle,linux/mips64,linux/mips64le
-          CACHE_FROM: type=gha,scope=frontend-${{ needs.frontend-base.outputs.typ }}
-          CACHE_TO: type=gha,scope=frontend-${{ needs.frontend-base.outputs.typ }}
+      -
+        name: Set cache attributes
+        if: "!startsWith(github.ref, 'refs/tags/v')"
+        run: |
+          # we don't want to cache the labs image
+          echo "CACHE_TO=" >>${GITHUB_ENV}
```

Comment:
For completion, let's clear `CACHE_FROM` as well.

Reply:
We don't want to clear cache-from actually but take the same one as previous

Reply:
It's a bit confusing right now but will be sorted with #3204